Jenkinsfile

```diff
@@ -14,13 +14,15 @@ pipeline {
 		steps {
                     parallel(
                         "Runtime Tests": { sh './contrib/vagrant/start.sh' }, 
-                         "K8s Tests": { sh './tests/k8s/start' } 
+                        "K8s Tests": { sh './tests/k8s/start' }
                     )
 	        }
         }
     }
     post {
         always {
+            sh './tests/copy_files'
+            archiveArtifacts "cilium-files.tar.gz"
             sh 'vagrant destroy -f'
```

Comment:
Nice, this makes the artefact available for download from Jenkins

https://jenkins.cilium.io/job/cilium/job/cilium/job/PR-1017/35/artifact/cilium-files.tar.gz